src/applications/rated-disabilities/components/NeedHelp.jsx

```diff
@@ -3,14 +3,13 @@ import { CONTACTS } from '@department-of-veterans-affairs/component-library/cont
 
 export default function NeedHelp() {
   return (
-    <va-need-help class="vads-u-margin-y--3" uswds="false">
+    <va-need-help class="vads-u-margin-y--3">
       <div slot="content">
         <p>
-          You can call us at{' '}
-          <va-telephone contact={CONTACTS.VA_BENEFITS} uswds="false" />. We’re
```

Comment:
There is a ticket/pr for this uswds work so I wouldnt update here its out of scope